containers-ordered-strict/src/Map/Ord.hs

```diff
@@ -4,26 +4,26 @@ module Map.Ord
        ( Map
        , Key
 
-       , empty
-       , singleton
-       , fromList
-
-       , null
-       , size
-       , member
-       , lookup
-       , lookupDefault
-
-       , toList
-       , keys
-       , elems
-
-       , insert
-       , insertWith
-       , adjust
-       , update
-       , delete
-       , alter
+       , M.empty
+       , M.singleton
+       , M.fromList
+
+       , M.null
+       , M.size
+       , M.member
+       , M.lookup
+       , M.findWithDefault
+
+       , M.toList
+       , M.keys
```

Comment:
`toList` and `keys` functions have different constraints in signature and in `Data.Map.Strict` function. So, again, unfortunately, you can't just export them from the qualified module 😞 These two functions should be written explicitly as before.